src/reducers/LoginReducer.jsx

```diff
@@ -31,6 +31,17 @@ const LoginReducer = (state = initialState, action) => {
         allCourses: action.courses,
         loggedIn: true,
       });
+    case 'CHANGE_SEMESTER': {
+      const isCourseMatch = course => course.courseId === action.courseId;
+      const index = state.plannedCourses.findIndex(isCourseMatch);
+      const course = state.plannedCourses.find(isCourseMatch);
+      course.semester = action.newSemester;
+      const updatedPlanned = state.plannedCourses;
+      updatedPlanned[index] = course;
+      return Object.assign({}, state, {
+        plannedCourses: updatedPlanned,
+      });
+    }
```

Comment:
So... the state that I need is in LoginReducer, which doesn't really make sense for actions that have nothing to do with logging in... Would it be better to split up receiving the state at login into a different reducer, or to just change the name of the file to encompass all the actions it handles?